bench/report.rb

```diff
@@ -50,7 +50,7 @@
     transaction_count: count,
     executed_at: Time.new.iso8601,
     'git.commit' => git_sha,
-    'git.date' => git_date && Time.parse(git_date).iso8601,
+    'git.date' => String(git_date).strip != '' && Time.parse(git_date).iso8601,
```

Comment:
Could you move this to a separate commit/PR please?

Reply:
Too late 😬